test/com/jambit/DatabaseConnectionTest.java

```diff
@@ -22,49 +21,47 @@
 
   @BeforeAll
   static void init() throws IOException, SQLException {
-    DatabaseConnection.changePropertyPath("config/appTest.properties");
+    DatabaseConnection.setPropertyPath("config/appTest.properties");
     databaseConnection = DatabaseConnection.getInstance();
   }
 
   @Test
   @Order(1)
-  public void insertTest() throws SQLException {
-    MoodEntries testValue =
-        new MoodEntries(0, (int) Math.round(Math.random() * 10), System.currentTimeMillis() / 1000);
-    databaseConnection.writeMoodEntries(testValue);
-    ArrayList<MoodEntries> dbOutputValues = databaseConnection.fetchAllMoodEntries();
-    assertEquals(dbOutputValues.get(dbOutputValues.size() - 1).time, testValue.time);
-    assertEquals(dbOutputValues.get(dbOutputValues.size() - 1).vote, testValue.vote);
+  public void writeIntoDatabase() throws SQLException {
+    MoodEntry expectedOutput =
+        new MoodEntry(0, (int) Math.round(Math.random() * 10), System.currentTimeMillis() / 1000);
+    databaseConnection.writeMoodEntry(expectedOutput);
+    ArrayList<MoodEntry> actualOutput = databaseConnection.fetchAllMoodEntries();
+    assertTrue(actualOutput.get(actualOutput.size() - 1).checkEquals(expectedOutput));
   }
 
   @Test
   @Order(2)
-  public void readTest() throws SQLException {
+  public void readOutOfDatabase() throws SQLException {
     long currentTime = System.currentTimeMillis() / 1000L;
-    int hours = (int) Math.round(Math.random() * 10 + 1);
-    long randomTime = currentTime - (hours * 60 * 60);
+    float hours = (int) Math.round(Math.random() * 10 + 1);
+    long randomTime = (long) (currentTime - (hours * 60 * 60));
 
-    ArrayList<MoodEntries> dbFilteredValues = databaseConnection.fetchMoodEntries(hours);
-    ArrayList<MoodEntries> dbFilterOfAllValues = new ArrayList<>();
+    ArrayList<MoodEntry> actualFilteredValues = databaseConnection.fetchMoodEntries(hours);
+    ArrayList<MoodEntry> expectedFilteredValues = new ArrayList<>();
     {
-      ArrayList<MoodEntries> dbAllValues = databaseConnection.fetchAllMoodEntries();
-      for (int i = 0; i < dbAllValues.size(); i++) {
-        if (dbAllValues.get(i).time <= currentTime && dbAllValues.get(i).time >= randomTime) {
-          dbFilterOfAllValues.add(dbAllValues.get(i));
-          dbAllValues.get(i);
+      ArrayList<MoodEntry> dbAllValues = databaseConnection.fetchAllMoodEntries();
```

Comment:
Get rid of generic namings like that! Allways have the context in mind: The moment you read "dbAllValues" could be everything out of a database! So I have to read the context and the more I have to read the more time I need to spent with your code...
better: "expected" or "expectedMoodEntries", then I'll check what Datatype this variable is I'm done: It's an list of MoodEntries you expect. Bam.